primitives/core/src/offchain/mod.rs

```diff
@@ -50,6 +52,7 @@ pub trait OffchainStorage: Clone + Send + Sync {
 
 /// A type of supported crypto.
 #[derive(Clone, Copy, PartialEq, Eq, Encode, Decode, RuntimeDebug, PassByEnum)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
```

Comment:
```suggestion
#[cfg_attr(feature = "std", derive(serde::Serialize, serde::Deserialize))]
```
You can do this to avoid adding extra, feature-gated import line.